cellprofiler/modules/identifytertiaryobjects.py

```diff
@@ -1,7 +1,10 @@
 # coding=utf-8
 
 """
-**Identify Tertiary Objects** identifies tertiary objects (e.g.,
+IdentifyTertiaryObjects
+=======================
+
+**IdentifyTertiaryObjects** identifies tertiary objects (e.g.,
 cytoplasm) by removing smaller primary objects (e.g. nuclei) from larger
 secondary objects (e.g., cells), leaving a ring shape.
 
```

Comment:
The line below this one has unnecessary `-----`. Please remove it.

Reply:
Also, remove the quotes around `"Measurements made by this module"` on L47 to fix the bad hyperlink.